src/index.js

```diff
@@ -2,3 +2,6 @@ export { default as v1 } from './v1.js';
 export { default as v3 } from './v3.js';
 export { default as v4 } from './v4.js';
 export { default as v5 } from './v5.js';
+export { default as REGEX } from './regex.js';
+export { default as version } from './version.js';
```

Comment:
Same discussion as for `REGEX` applies here as well… `UUIDVersion()`? Looks a bit weird but would be consistent with `randomUUID()`-style naming…

Reply:
[Just deleted a long spiel about how to rename things.]

Out of respect for @awwit, let's not go down the renaming rabbit hole in this PR.  We've already agreed to push this work to `v9` so I think it'd be better to stick with the current custom of terse names for the moment (`version`, `validate`, and `REGEX`), and open a new issue for the renaming discussion.   We could even put something in the README (on `master`) asking people to chime in with their thoughts to help generate discussion.

Reply:
OK, fine for me.

What I would however like to see before we merge is some initial documentation of the new methods in the `README_js.md`. Could you add that @awwit ? I'm happy to merge a first iteration into a new development branch then.